front/src/applications/operationalStudies/hooks/useLazySimulateTrains.ts

```diff
@@ -0,0 +1,107 @@
+import { useRef, useEffect, useCallback, useState } from 'react';
+
+import type {
+  LightRollingStockWithLiveries,
+  SimulationSummaryResult,
+} from 'common/api/osrdEditoastApi';
+import type { TimetableItemWithDetails } from 'modules/trainschedule/components/Timetable/types';
+import type { TimetableItemId, TimetableItemWithTimetableId } from 'reducers/osrdconf/types';
+import { useAppDispatch } from 'store';
+
+import formatTimetableItemSummaries from '../helpers/formatTimetableItemSummaries';
```

Comment:
I may be doing it wrong, but the imports are broken when I try to test it locally

```suggestion
import formatTimetableItemSummaries from 'modules/simulationResult/helpers/formatTimetableItemSummaries';
```

Reply:
Not sure what would cause this, relative imports are supposed to work fine and CI doesn't show any error.

Reply:
let's ignore it then